src/components/avatar/avatar.tsx

```diff
@@ -0,0 +1,21 @@
+// demo:<Avatar email="admin@gmail.com"/>
+
+const Avatar=({email=''})=>{
+    const styleBox={
+        width:'100%',
+        height:'100%',
+        background:'#00FF7F',
```

Comment:
chỗ này set cứng luôn hả Thành, đôi khi nó là random mà :vvvvvvv có cái Math.random() đó Thành

Reply:
thầy bảo chuyển qua sư dụng scss lun

Reply:
mình định cho nó một cái div bọc ở ngoài, rồi kích thước sẽ theo kích thước của cái div, tại có lúc nó to có lúc nó nhỏ ak